composables/useKyVerified.ts

```diff
@@ -0,0 +1,56 @@
+import { computed } from 'vue'
+import { useBase } from '~/composables/useBase'
+import { KYBStatusEnum } from '~/plugins/weavr-multi/api/models/identities/corporates/enums/KYBStatusEnum'
+import { KYCStatusEnum } from '~/plugins/weavr-multi/api/models/identities/consumers/enums/KYCStatusEnum'
+
+export function useKyVerified() {
+    const {
+        isCorporate,
+        isConsumer,
+        corporate,
+        consumer,
+        corporateKyBStatus,
+        consumerKyC,
+        consumerKyCStatus,
+        stores,
+    } = useBase()
+
+    const hasAlert = computed(() => showKybAlert.value || showKycAlert.value)
+
+    const showKybAlert = computed(() => {
+        if (!isCorporate.value) {
+            return false
+        }
+
+        if (corporate.value && corporateKyBStatus.value) {
+            return corporateKyBStatus.value !== KYBStatusEnum.APPROVED
+        }
+
+        return false
```

Comment:
```suggestion
        if (corporate.value && corporateKyBStatus.value) {
            return corporateKyBStatus.value !== KYBStatusEnum.APPROVED
        }

        return false
```
Would the removal of that if statement produce the same result?

Reply:
> Would the removal of that if statement produce the same result?

works as an early return, so computed won't even run for consumer @oleggrech-t which i think is ok